### ISSUE TITLE
Notice: Undefined index.php

### ISSUE THREAD

**Issue by leakedsources** [binarymaster/3WiFi#68]
Hi
I tried both php versions 5 and 7 but still have problems
Please give me the php version you used and more details

```
Notice: Undefined index: uselocation in index.php on line 149
Notice: Undefined index: lat in index.php on line 151
Notice: Undefined index: lon in index.php on line 152
Notice: Undefined index: rad in index.php on line 153
Notice: Undefined index: lat in index.php on line 155
Notice: Undefined index: lon in index.php on line 156
Notice: Undefined index: rad in index.php on line 157
```

**Comment by leakedsources** [comment]
no one is answer?

**Comment by mitya12342** [comment]
These are notices about non existing cookies, everything completely fine about them (besides of clogging logs).

I got script running on windows with [XAMPP](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.38/) of PHP 5.6.38 Apache 2.4.34 MariaDB 10.1.36.
(no luck with php7 linux deployment for now though)
Things to beware of:
- I had to change php.ini options
```
short_open_tag = On
display_errors = Off
```
- Known php modules used (included with xampp php) are mysqli, curl and bcmath
- Ad-blockers love to eat yandex maps
- `$HTTP_RAW_POST_DATA` is deprecated and will be removed in 7.0.0. Replacing by `file_get_contents("php://input")` seem to work 
- .htaccess seems to go funky in newer apache versions. Easy fix is to replace `index.php?redir=$1` by `index.php?page=$1`

Had some fun trying to run this script correctly 
My only problem for now is randomly hanging geolocation script which is not crucial in my case
Good luck!
